### PR TITLE
docs: migrate from Duende guide

### PR DESCRIPTION
**Pull request by sixhobbits**
depends on:

- https://github.com/FusionAuth/fusionauth-import-scripts/pull/9
- https://github.com/FusionAuth/fusionauth-contrib/pull/11

### CODE REVIEW

File: site/docs/v1/tech/migration-guide/duende.adoc [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+[NOTE.note]
+====
+The social account linking functionality will only work with FusionAuth version 1.28 or above. The `fusionauth_client` library must be >= 1.28.
+====
+
+When you run the script, you should get an output similar to the following:
+
+[source,shell,title=Import script output]
+----
+$ ruby ./import.rb -f http://localhost:9011 -k '...' -u user-data.json
```
Comment (ColinFrick) [inline site/docs/v1/tech/migration-guide/duende.adoc:289]: Could you add `-t '...'` to the command. I missed this on my first try, which caused the import.rb to crash with a non-descriptive error message:
```
Traceback (most recent call last):
	2: from ./import.rb:240:in `<main>'
	1: from ./import.rb:156:in `import'
./import.rb:156:in `+': no implicit conversion of nil into String (TypeError)
```

**Review by ColinFrick — CHANGES_REQUESTED** [review]

**Comment by rideam** [comment]
@ColinFrick I pushed updates

**Comment by mooreds** [comment]
@ColinFrick can you please re-review?

**Review by ColinFrick — APPROVED** [review]
LGTM

**Comment by mooreds** [comment]
Closed under https://github.com/FusionAuth/fusionauth-site/pull/2257

**Pull request closed without merging**